src/signals-win.c

```diff
@@ -326,18 +326,53 @@ volatile HANDLE hBtThread = 0;
 static DWORD WINAPI profile_bt( LPVOID lparam )
 {
     // Note: illegal to use jl_* functions from this thread
-
     TIMECAPS tc;
     if (MMSYSERR_NOERROR!=timeGetDevCaps(&tc, sizeof(tc))) {
         fputs("failed to get timer resolution",stderr);
         hBtThread = 0;
         return 0;
     }
+
+	//Precision timer
```

Comment:
Please use multiples of 4 spaces for indentation rather than tabs for consistency with the rest of the repository.